src/cli-validator/utils/fixProblems.js

```diff
@@ -0,0 +1,229 @@
+const each = require('lodash/each');
+const fs = require('fs');
+const snakeCase = require('snake-case');
+
+// this function prints all of the output
+module.exports = function print(
+  results,
+  originalFile,
+  errorsOnly,
+  validFile,
+  doFixProblemsNewFile
+) {
+  const types = errorsOnly ? ['errors'] : ['errors', 'warnings'];
+
+  const originalJSON = JSON.parse(originalFile);
+
+  const paramsToDelete = {};
+  const paramsToPrepend = {};
+  const newPathNames = {};
+  const definitionsToDelete = {};
+
+  const markForDeletion = path => {
```

Comment:
Since these are effectively method declarations, I think you should add a comment above each that briefly describes what they do.  For example, I think you expect `path` to the be the path to a parameter -- it would wrong to think that this method could mark an arbitrary element for deletion.